src/main/fc/config.c

```diff
@@ -447,6 +447,10 @@ void createDefaultConfig(void)
 #endif
 #endif
 
+#ifdef OSD
+    featureSet(FEATURE_OSD);
```

Comment:
I wonder if the same could not be done on TARGET level using
`#define DEFAULT_FEATURES`

Reply:
It could be done at the target level, but this way enables it for all current and future targets without any code duplication. The only board I can think of where the OSD support is compiled but we might want to disable by default is the MATEKF405, since there's a variant with OSD and another one without it. Since that's gonna be the exception rather than the common case, I'd prefer do it the other way around i.e.,

```#if defined(OSD) && !defined(OSD_FEATURE_NOT_DEFAULT)```

That doesn't require any additional definitions if you want the OSD to be enabled by default (all but one board right now) and if you don't want it enabled by default you can just define `OSD_FEATURE_NOT_DEFAULT`.

Reply:
We have `DEFAULT_FEATURES` infrastructure for a reason - to enable target-specific features without messing with core code. 

Bear is mind that OSD is not dependent on having MAX7456 chip onboard - it may use external hardware via `displayPort`. In this case having `#define OSD` doesn't mean we want the feature enabled explicitly since it may be dependent on non-connected hardware.

Reply:
Ok, let's do it via `DEFAULT_FEATURES`. Which boards besides OMNIBUS variants we'd like to have OSD enabled by default?